### ISSUE TITLE
Node Sass does not yet support your current environment

### ISSUE THREAD

**Issue by danielcharrua** [understrap/understrap-child#275]
Hi, I have installed Node v15 and I can’t run `gulp watch` because this error:

Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (88)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.11.0

It seems that we can’t create the js/css bundle with node v15

Any idea on how to fix this?

**Comment by LukaszJaro** [comment]
you just need to update sass to latest v5.0.0

**Comment by bacoords** [comment]
Anyone having this issue: please pull the latest version that uses postcss in place of node-sass.

**Comment by butterp1** [comment]
Hi,
I had to update a quickly theme still on understrap-child 0.9.5. Copy installed on localhost.

Recently I had to update node and npm to v16.8.0 and 7.21.0 for another project, so
I wasn't surprised when gulp watch didn't work on this project. 

I tried a clean install of understrap-child v0.9.6. NPM install threw the same error as earlier posts:
`Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (93)`

After much poking about, I edited package.json and updated gulp-sass to:
`"gulp-sass": "^5.0.0", `

This removes the requirement for "node-sass": "^4.8.3" (in package-lock.json).

NPM install worked fine now. Gulp watch was still giving trouble though:

```
gulp-sass 5 does not have a default Sass compiler; please set one yourself.
Both the `sass` and `node-sass` packages are permitted.
```

So... 
`npm i node-sass `
(for version 6.0.1)

And in gulpfile.js:
`var sass = require('gulp-sass')(require('node-sass'));`

NPM install worked fine now, and gulp watch - just thought I should share. I doubt it did it the best way, but it worked.
